edge-modules/iotedge-diagnostics-dotnet/src/GetSocket.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Diagnostics
```

Comment:
I think this should be similar to Edge Agent and Edge Hub projects, it should start with Microsoft.Azure.Devices.Edge.

Reply:
Its not in that namespace though. I don;t think this matters that much